contracts/src/v0.8/dev/DerivedPriceFeed.sol

```diff
@@ -18,35 +18,36 @@ import "../interfaces/AggregatorV2V3Interface.sol";
  * Quote Address: 0x5498BB86BC934c8D34FDA08E81D444153d0D06aD
  * Decimals: 18
  *
- * Chainlink Data Feeds can be used in combination to derive denominated price pairs in other currencies.
- * If you require a denomination other than what is provided, you can use two data feeds to derive the pair that you need.
- * For example, if you needed a LINK / FTM price, you could take the LINK / USD feed and the FTM / USD feed and derive LINK / FTM using division.
+ * Chainlink Data Feeds can be used in combination to derive denominated price pairs in other
+ * currencies.
+ *
+ * If you require a denomination other than what is provided, you can use two data feeds to derive
+ * the pair that you need.
+ *
+ * For example, if you needed a LINK / FTM price, you could take the LINK / USD feed and the
+ * FTM / USD feed and derive LINK / FTM using division.
  * (LINK/USD)/(FTM/USD) = LINK/FTM
  */
-contract DerivedPriceFeed is AggregatorV2V3Interface {
+contract DerivedPriceFeed is AggregatorV3Interface {
   uint256 public constant override version = 0;
 
-  uint8 public override decimals;
-  int256 public override latestAnswer;
-  uint256 public override latestTimestamp;
-  uint256 public override latestRound;
-
-  mapping(uint256 => int256) public override getAnswer;
-  mapping(uint256 => uint256) public override getTimestamp;
-  mapping(uint256 => uint256) private getStartedAt;
-
-  AggregatorV3Interface public base;
-  AggregatorV3Interface public quote;
+  AggregatorV3Interface public immutable BASE;
+  AggregatorV3Interface public immutable QUOTE;
```

Comment:
Is there ever a situation where we'd want to set these to something else? Or are these truly a "set it and forget it" thing?

Reply:
You mean like change them after deployment? I don't think so, if we wanted to make them changeable would probably point to a proxy here

Reply:
Yeah, I meant after deployment. Sounds good 👍